appeals/web/src/server/appeals/appeal-details/interested-party-comments/interested-party-comments.controller.js

```diff
@@ -1,70 +1,84 @@
 import logger from '#lib/logger.js';
-import { interestedPartyCommentsPage } from './interested-party-comments.mapper.js';
+import {
+	interestedPartyCommentsPage,
+	viewInterestedPartyCommentPage
+} from './interested-party-comments.mapper.js';
 import * as interestedPartyCommentsService from './interested-party-comments.service.js';
 
 /**
  *
  * @param {import('@pins/express/types/express.js').Request} request
  * @param {import('@pins/express/types/express.js').RenderedResponse<any, any, Number>} response
  */
-const renderInterestedPartyComments = async (request, response) => {
+export const renderInterestedPartyComments = async (request, response) => {
 	const { errors, currentAppeal } = request;
 	const paginationParameters = {
 		pageNumber: 1,
 		pageSize: 1000
 	};
 
-	if (
-		currentAppeal &&
-		currentAppeal.appellantCaseId !== null &&
-		currentAppeal.appellantCaseId !== undefined
-	) {
-		try {
-			const [awaitingReviewComments, invalidComments, validComments] = await Promise.all([
-				interestedPartyCommentsService.getInterestedPartyComments(
-					request.apiClient,
-					currentAppeal.appealId,
-					'awaiting_review',
-					paginationParameters.pageNumber,
-					paginationParameters.pageSize
-				),
-				interestedPartyCommentsService.getInterestedPartyComments(
-					request.apiClient,
-					currentAppeal.appealId,
-					'invalid',
-					paginationParameters.pageNumber,
-					paginationParameters.pageSize
-				),
-				interestedPartyCommentsService.getInterestedPartyComments(
-					request.apiClient,
-					currentAppeal.appealId,
-					'valid',
-					paginationParameters.pageNumber,
-					paginationParameters.pageSize
-				)
-			]);
+	if (currentAppeal?.appellantCaseId == null) {
+		return response.status(404).render('app/404.njk');
+	}
 
-			const mappedPageContent = await interestedPartyCommentsPage(
-				currentAppeal,
-				awaitingReviewComments,
-				validComments,
-				invalidComments
-			);
+	try {
+		const [awaitingReviewComments, invalidComments, validComments] = await Promise.all([
+			interestedPartyCommentsService.getInterestedPartyComments(
+				request.apiClient,
+				currentAppeal.appealId,
+				'awaiting_review',
```

Comment:
I know this was pre-existing, but these could reference `APPEAL_REPRESENTATION_STATUS` from `packages/appeals/constants/common.js` instead of hardcoding